### DISCUSSION TITLE
FastAPI-Users appears to not work with Atlas Serverless

### DISCUSSION THREAD

**Discussion by igordertigor** [fastapi-users/fastapi-users#1375] [Q&A]
I'm using FastAPI-Users with mongodb. Locally, everything works smoothly and I like the simplicity of setting this up. Yet, when wiring things up with my Atlas Serverless Instance, I get
```
pymongo.errors.OperationFailure: collation not allowed in this atlas tier, full error: {'ok': 0, 'errmsg': 'collation not allowed in this atlas tier', 'code': 8000, 'codeName': 'AtlasError'}
```
The full traceback is obviously much longer, but before posting it, I wanted to get an idea if others are facing this issue too. Or has nobody been playing with the Atlas Serverless offering yet?

**Comment by frankie567** [comment]
From what I read [here](https://www.mongodb.com/docs/atlas/reference/serverless-instance-limitations/#operational-limitations-and-considerations), Atlas Serverless does not support collations.

We're using collations to build a case-insensitive index for emails: https://github.com/fastapi-users/fastapi-users-db-beanie/blob/eedd1039cd8d3aece0f49fa1a90d9649ca80d1ee/fastapi_users_db_beanie/__init__.py#L23-L30

What you can try is to disable this by overloading the settings:

```py
class User(BeanieBaseUser, Document):
    class Settings:
        indexes = [
            IndexModel("email", unique=True),
        ]
```
